### ISSUE TITLE
Missing help documentation for parameter SslProtocol of Invoke-WebRequest and Invoke-RestMethod cmdlets

### ISSUE THREAD

**Issue by ScriptAutomate** [PowerShell/PowerShell-Docs#1866]
Issue Details
-----------------
_Note: Directly related to [PR #5329 in PowerShell/PowerShell](https://github.com/PowerShell/PowerShell/pull/5329)_

Definition and summary information, that can be used for creating the documentation, could be taken from the [commits themselves](https://github.com/PowerShell/PowerShell/commit/ecf0f8c15189b4727fabb3df38867de9cd4679a4), which I have pretty much copy/pasted below.

Documentation for `-SslProtocol` is missing for: 

- [Invoke-RestMethod](https://github.com/PowerShell/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Utility/Invoke-RestMethod.md)
- [Invoke-WebRequest](https://github.com/PowerShell/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Utility/Invoke-WebRequest.md)

Default value is `Default`

Accepted/Restricted values at the moment are based on available values for the `Microsoft.PowerShell.Commands.WebSslProtocol` type: 
- `Default`
- `Tls`
- `Tls11`
- `Tls12`

### Potential definitions:
Could be simple:
```
Sets the TLS/SSL protocol used by the Web Cmdlet.
```
Or, if after each new value is made available for the `Microsoft.PowerShell.Commands.WebSslProtocol` type, start with the following description (and update over time):
```
Sets the TLS/SSL protocol used by the Web Cmdlet. The following values can be used: Default, Tls, Tls11, or Tls12.

Default means no SSL protocol will be set and the system defaults will be used. Tls is the TLS 1.0 security protocol. The TLS protocol is defined in IETF RFC 2246. Tls11 is the TLS 1.1 security protocol. The TLS protocol is defined in IETF RFC 4346. Tls12 is the TLS 1.2 security protocol. The TLS protocol is defined in IETF RFC 5246
```
May look something like this in the parameter YAML:
```yaml
Type: WebSslProtocol
Parameter Sets: (All)
Aliases:

Required: False
Position: Named
Default value: Default
Accept pipeline input: False
Accept wildcard characters: False
```
_Note: I have not tested whether values can be passed in via pipeline, or whether it is available for all parameter sets._ 

Version(s) of document impacted
------------------------------
- [x] Impacts 6 document
- [ ] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not selecting all version of documents
----------------------------------------------------
- [x] The documented feature was introduced in selected version of PowerShell
- [x] This issue only shows up in selected version of the document

**Comment by ScriptAutomate** [comment]
In Progress

**Comment by markekraus** [comment]
Documentation was added in #1870

**Comment by zjalexander** [comment]
Thanks @markekraus ! Looks like this issue can be closed as it is resolved via #1870 :)
